parser.go

```diff
@@ -433,6 +441,50 @@ func (p *parser) parseInfo() error {
 	return nil
 }
 
+func (p *parser) parseMapping() error {
+	fileTree, err := goparser.ParseFile(token.NewFileSet(), p.MainFilePath, nil, goparser.ParseComments)
+	if err != nil {
+		return fmt.Errorf("can not parse mapping from general API information: %v", err)
+	}
+
+	if fileTree.Comments != nil {
+		for i := range fileTree.Comments {
+			for _, comment := range strings.Split(fileTree.Comments[i].Text(), "\n") {
+				attribute := strings.ToLower(strings.Split(comment, " ")[0])
+				if len(attribute) == 0 || attribute[0] != '@' {
+					continue
+				}
+				value := strings.TrimSpace(comment[len(attribute):])
+				if len(value) == 0 {
+					continue
+				}
```

Comment:
I thought about that originally. But everything in `parseInfo` is part of the OpenAPI spec except this. That's why I moved it to its own function. We could rename it to something more generic in case there's additional no spec items we add in there.

Otherwise I'm open to folding it in also.

Reply:
Gotcha. Yeah we may want to rename `parseInfo`, since it's already beyond the scope of parsing the `info` object within the open API spec. I could see us breaking the function up into multiple pieces to get some separation of concerns, but hopefully we'd do so without reproducing too much of its boilerplate.